coderd/database/migrations/000186_user_favorite_workspaces.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE ONLY workspaces
+ADD COLUMN favorite_of uuid DEFAULT NULL;
+COMMENT ON COLUMN workspaces.favorite_of IS 'FavoriteOf contains the UUID of the workspace owner if the workspace has been favorited.';
```

Comment:
self-review: just storing a boolean value here doesn't provide enough context to the query to be able to know how to sort favorites.